app/renderer/components/Inspector/RecordedActions.js

```diff
@@ -66,7 +65,7 @@ class RecordedActions extends Component {
             <Button
               onClick={toggleShowBoilerplate}
               icon={<ExportOutlined/>}
-              type={boilerplateType}
+              type={showBoilerplate ? BUTTON_PRIMARY : BUTTON_DEFAULT}
```

Comment:
we could still reduce the duplication by having that logic in a function, for example:
```
function getButtonType (showBoilerplate) {
    return showBoilerplate ? BUTTON_PRIMARY : BUTTON_DEFAULT;
}
```